force-app/main/default/lwc/miscPermissionBasedUI/miscPermissionBasedUI.js

```diff
@@ -0,0 +1,10 @@
+import { LightningElement } from 'lwc';
+
+// import the permission from userPermission (standard Salesforce permissions) or
+// customPermission (custom org-defined permissions)
+import hasAccessRestrictedUI from '@salesforce/customPermission/accessRestrictedUIPermission';
+
+export default class MiscPermissionBasedUI extends LightningElement {
+    // surface imported permission to HTML template
+    isRestrictedUIAccessible = hasAccessRestrictedUI;
```

Comment:
The preferred way is to use a getter, and not a direct assignment.

Reply:
I copied this from another recipe. Opened #323 to search and squash any other instances of direct assignment to props where it would be safer to implement as getter/setter. 

Otherwise, fixed here.

Reply:
This also aligns with the recommended way, as documented [here](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.create_get_permissions).